app/src/main/java/com/example/myapplication/ManaListItem.java

```diff
@@ -0,0 +1,40 @@
+package com.example.myapplication;
+
+public class ManaListItem {
+
+    private int manaImg;
+    private String manaType;
+    private String manaPrice;
+
+
+
+    public ManaListItem(int manaImg, String manaType, String manaPrice) {
+        this.manaImg = manaImg;
```

Comment:
what is the purpose of manImg ? Why we need it for ?

Reply:
It contains the picture of the card for the pageViewer
Pita, Half Pita, Lafa